homeassistant/components/zwave_js/config_flow.py

```diff
@@ -667,30 +692,15 @@ async def async_step_configure_addon_user(
             # Network already exists, go to security keys step
             return await self.async_step_configure_security_keys()
 
-        usb_path = self.usb_path or ""
-
-        schema: VolDictType = {
-            vol.Required("network_type", default=""): vol.In(
-                [NETWORK_TYPE_NEW, NETWORK_TYPE_EXISTING]
-            )
-        }
-
-        if not self._usb_discovery:
-            try:
-                ports = await async_get_usb_ports(self.hass)
-            except OSError as err:
-                _LOGGER.error("Failed to get USB ports: %s", err)
-                return self.async_abort(reason="usb_ports_failed")
-
-            schema = {
-                vol.Required(CONF_USB_PATH, default=usb_path): vol.In(ports),
-                **schema,
-            }
-
-        data_schema = vol.Schema(schema)
-
         return self.async_show_form(
-            step_id="configure_addon_user", data_schema=data_schema
+            step_id="network_type",
+            data_schema=vol.Schema(
+                {
+                    vol.Required("network_type", default=""): vol.In(
```

Comment:
I don't understand the benefit of the default value since this is a required input.

Reply:
Without the default it auto selects the first option

Reply:
Isn't that a frontend behavior that is weird?

Reply:
I don't know if it's on purpose but saw `default=""` in some other config flows and thought that this is the way.
Also checked the frontend and it seems pretty explicit https://github.com/home-assistant/frontend/blob/9ec38c7dd90f2a067c754ee04fcb1a67667b16e1/src/components/ha-form/compute-initial-ha-form-data.ts#L36

Reply:
It's not documented and I didn't know about it.

https://developers.home-assistant.io/docs/data_entry_flow_index#defaults--suggestions

We can keep it here, but it would be good to document it if it's an expected feature of the interface.

Reply:
Intended or not, it has worked like that for a few years by the looks of it, so it would be a breaking change to fix it.